src/utils.cpp

```diff
@@ -443,7 +443,7 @@ string * TLVstore::tlvRecordsToContainer()
 
 string TLVstore::get(string type)
 {
-    return tlv.at(type);
+    return tlv[type];
```

Comment:
@polmr What's the goal of this change? Doesn't `std::map<key,value>::at()` compile in Android?
The reason of using `at()` instead of the operator `[ ]` is because the second one automatically creates the item in the map for the requested key in case it doesn't exist. And that's not the expected behaviour of a `get()`